.travis.yml

```diff
@@ -61,6 +62,7 @@ jobs:
         - ddev validate metadata
         - ddev validate service-checks
         - ddev test --cov
+        - ddev env test
```

Comment:
I guess it's missing `--clean` here?

Reply:
No b/c https://github.com/DataDog/integrations-core/pull/3896/files#diff-55c0b31a1b825f162922bf2e5f22a705R46